src/Microsoft.Framework.Runtime/DependencyManagement/LockFileFormat.cs

```diff
@@ -243,7 +243,7 @@ private IPackageFile ReadPackageFile(string property, JToken json)
         private JProperty WritePackageFile(IPackageFile item)
         {
             var json = new JObject();
-            return new JProperty(item.Path, new JObject());
+            return new JProperty(item.Path.Replace('\\', '/'), new JObject());
```

Comment:
nit: we have at least two places have codes to unify the separator. should be have common code?